tox.ini

```diff
@@ -32,14 +32,26 @@ deps = https://github.com/willkg/dennis/archive/aca57f6d.zip
 commands = dennis-cmd lint --errorsonly locale/
 
 [testenv:docker]
-commands = 
-    docker-compose -f docker-compose.yml -f scripts/docker-compose.travis.yml up -d
-    docker exec -i kuma_web_1 make compilejsi18n collectstatic
+whitelist_externals =
+    docker
+    docker-compose
+    make
+    sleep
+setenv =
+    COMPOSE_FILE = docker-compose.yml:scripts/docker-compose.travis.yml
+    COMPOSE_PROJECT_NAME = kumatox
+commands =
+    make build-base VERSION=latest
+    docker-compose build mysql
```

Comment:
I think, we can just pass the ``--build`` parameter to the ``docker-compose up``.
like ``docker-compose up --build -d mysql``